anvil_schema/fss_index_data_dictionary/fss_index_data_dictionary.json

```diff
@@ -0,0 +1,888 @@
+          "datatype": "string", 
+          "array_of": true, 
+          "required": false
+        }
+      ],
+      "primaryKey": ["donor_id"],
+      "partitionMode": "none",
+      "datePartitionOptions": null,
+      "intPartitionOptions": null,
+      "reqiured": false
+    },
+    {
+      "name": "anvil_file",
+      "description": "Information for files associated with the study.",
+      "columns": [
+        {
+          "name": "file_id", 
+          "description": "Unique identifier for the file this record represents.",
+          "datatype": "string", 
+          "array_of": false, 
+          "required": true
+        },
+        {
+          "name": "data_modality", 
+          "description": "Data modality describes the biological nature of the information gathered as the result of an Activity, independent of the technology or methods used to produce the information.",
+          "datatype": "string", 
+          "array_of": true, 
+          "required": false},
+        {
+          "name": "file_format", 
+          "description": "An indication of the format of an electronic file; include the full file extension including compression extensions. Usually aligns with file extension (e.g. bam, sam, text, csv, etc.)",
+          "datatype": "string", 
+          "array_of": false, 
+          "required": false
+        },
+        {
+          "name": "file_size", 
+          "description": "Size of file in megabytes.",
+          "datatype": "integer", 
+          "array_of": false, 
+          "required": false
+        },
+        {
+          "name": "file_md5sum", 
+          "description": "md5 checksum for the file",
+          "datatype": "string", 
+          "array_of": false, 
+          "required": false
+        },
+        {
+          "name": "reference_assembly", 
+          "description": "A reference to the collection of sequences taken as the standard for a given organism. May be defined by https://www.ncbi.nlm.nih.gov/grc.",
+          "datatype": "string", 
+          "array_of": true, 
+          "required": false
+        },
+        {
+          "name": "file_name",
+          "description": "The name of the file.",
+          "datatype": "string", 
+          "array_of": false, 
+          "required": false
+        },
+        {
+          "name": "file_ref", 
+          "description": "The fully qualified path to the file.",
+          "datatype": "fileref", 
+          "array_of": false, 
+          "required": false
+        },
+        {
+          "name": "is_supplementary", 
+          /// this could use a better definition, but isn't necessarily worse than some of the others....
+          "description": "Whether a file is considered supplementary or not.",
```

Comment:
Indicates whether a file is considered supplementary or not, where supplementary is defined as a file that cannot be directly or indirectly tied back to a biosample record.